lisa/sut_orchestrator/azure/common.py

```diff
@@ -118,6 +118,7 @@ class SharedImageGallerySchema:
     image_gallery: str = ""
     image_definition: str = ""
     image_version: str = ""
+    hyperv_generation: str = "1"
```

Comment:
Is there any properties on shared image gallery? Let's check if this value can be gotten from Azure, instead of setting in LISA.

Reply:
_parse_shared_gallery_image will return type SharedImageGallerySchema, so I add the property for hyperv_generation